dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryShimTest.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2023 IBM Corporation and others.
```

Comment:
Is the date correct here? (It could be if it's mostly a copy of another file with an original date of 2022, but otherwise it should be 2023)